### PR TITLE
lets publish it ASAP

### PR DESCRIPTION
**Pull request by naumoff**
lets do it!

### CODE REVIEW

**Comment by testmailrest** [comment]
I do not think so!

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ Andrey Naumoff
 php web developer
 
 Who is super human?
+Who is not?
```
Comment (testmailrest) [inline README.md]: Why not?

File: .gitignore [hunk 2]
```diff
@@ -14,4 +14,5 @@ bootstrap/cache/
 # Rocketeer PHP task runner and deployment package. https://github.com/rocketeers/rocketeer
 .rocketeer/
 
+# I decided to comment it!
 .idea/*
```
Comment (testmailrest) [inline .gitignore:18]: it is bad idea!

Comment (naumoff) [reply .gitignore:18]: no it is good!

File: README.md (hunk 1 above)
Comment (naumoff) [reply README.md]: because

**Pull request merged**